seacrowd/sea_datasets/indonesiannmt/indonesiannmt.py

```diff
@@ -0,0 +1,188 @@
+                    "lang_2": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text_to_text.features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        # ex para: indonesiannmt-ind_jav_source OR indonesiannmt-ind_jav_seacrowd_t2t
+        if self.config.name in ['indonesiannmt_source', 'indonesiannmt_seacrowd_t2t']:
+            path = dl_manager.download_and_extract(_URLS['ind_jav'])
+        else:         
+            target = '_'.join(self.config.name.split('-')[1].split('_')[:2])
+            url = _URLS[target]
+            path = dl_manager.download_and_extract(url)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": path,
+                    "split": "train",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        with open(filepath, encoding="utf-8") as f:
+            flag = True  
+            if self.config.schema == "source":
+                for counter, row in enumerate(f):
+                    if flag:
+                        src, tgt = row.split('\t')
+                        tgt = tgt.strip()
+                        flag = False
+                    else:
+                        if row.strip() != "":
+                            yield(
+                                counter,
+                                {
+                                    "id": str(counter),
+                                    "text_1": row.split('\t')[0].strip(),
+                                    "text_2": row.split('\t')[1].strip(),
+                                    "lang_1": src,
+                                    "lang_2": tgt,
+                                }
+                            )
+            else:
+                for counter, row in enumerate(f):
+                    if flag:
+                        src, tgt = row.split('\t')
+                        tgt = tgt.strip()
+                        flag = False
+                    else:
+                        if row.strip() != "":
+                            yield(
+                                counter,
+                                {
+                                    "id": str(counter),
+                                    "text_1": row.split('\t')[0].strip(),
+                                    "text_2": row.split('\t')[1].strip(),
+                                    "text_1_name": src,
```

Comment:
Let's use the 3-letter ISO code of the language, e.g., `"ind"` instead of `"Indonesian"`.